packages/tx/src/legacyTransaction.ts

```diff
@@ -296,13 +296,7 @@ export default class Transaction extends BaseTransaction<Transaction> {
     this._validateHighS()
 
     try {
-      return ecrecover(
-        msgHash,
-        v!,
-        bigIntToUnpaddedBuffer(r!),
-        bigIntToUnpaddedBuffer(s!),
-        this.supports(Capability.EIP155ReplayProtection) ? this.common.chainId() : undefined
```

Comment:
Please do not do any changes to the transaction library in this PR, this is out of the scope of the task and by far too dangerous.